source/main/modules/export/run-pandoc.js

```diff
@@ -47,6 +47,12 @@ module.exports = async function (options) {
     throw new Error(trans('system.error.no_xelatex_message'), trans('system.error.no_xelatex_title'))
   }
 
+  // Include CSL library if exist
+  let bibliography = ''
+  if (global.config.get('export.cslLibrary') && isFile(options.cslLibrary)) {
```

Comment:
Just saw this, and that it's my fault: If there is a cslLibrary in the option, we should use that one, else if there is a global config value for that, use that, and only if both fail we shouldn't use anything.

The same holds true for the bibliography. Sorry, was my mistake, but I didn't have the time to track down the options cslLibrary (and also cslStyle where I made that mistake as well)!

Reply:
IMO, you should copy the default values to user config dir, and stick the app with it (aka. update config only in that json without touch default) to maintain hierarchy, instead of messing with if-else clause checking method all around codebase.

Reply:
BTW, AFAIK from the code, cslLibrary is the path to bibliography, which is fuzzing.